test/onnx/expect/TestFindMismatch_ONNX.test_pretty_print_tree_visualizes_mismatch.expect

```diff
@@ -0,0 +1,19 @@
+==================================== Tree: =====================================
+5 X   __2 X    __1 ✓
```

Comment:
A graph! That's lovely

Reply:
How was it done?

Reply:
A very naive implementation in `GraphInfoPrettyPrinter` class.